backend/controllers/EventController.js

```diff
@@ -20,9 +20,10 @@ function addEvent(req, res, next) {
 function updateEvent(req, res, next){
   /* Validation here */
 
-  const updateOptions = { new: true };
+  const updateOptions = { new: true};
 
-  Event.findByIdAndUpdate(req.params.id,{ $set: req.body }, updateOptions)
+  //Tried to just owner can update 
+  Event.findByIdAndUpdate( {'owner' : req.params.id},{ $set: req.body }, updateOptions)
```

Comment:
Is this correct? It seems like its already **findById**, why we pass **json object** into it? (Just intuition, no testing I'm afraid)